app/assets/stylesheets/admin/loans/timeline/_table.scss

```diff
@@ -1,4 +1,5 @@
 @import "../../colors";
+@import "../../variables";
```

Comment:
I'd ideally like to get away from importing variables and colors. Fixing the underlying bug that requires this as a fix is part of our backlog.

Reply:
Amen!